.github/workflows/semantic-pr.yml

```diff
@@ -132,29 +113,14 @@ jobs:
           # merge commit, and it's easy to commit this by mistake. Enable this option
           # to also validate the commit message for one commit PRs.
           validateSingleCommit: true
-
-  comment-on-pr:
-    runs-on: ubuntu-20.04
-    needs: check-semantic-pr
-    if: always()
-    env:
-      STATUS: ':heavy_check_mark:'
-      check-type: Semantic PR check
-      check-documentation: See [instructions on formatting your commit and pull request titles](https://github.com/magma/magma/wiki/Contributing-Code#pull-request-and-commit-message-title-are-following-conventional-commits-format).
-    steps:
-      - if: needs.check-semantic-pr.result == 'failure'
-        run: echo 'STATUS=:x:' >> $GITHUB_ENV
-      - uses: peter-evans/find-comment@b657a70ff16d17651703a84bee1cb9ad9d2be2ea # pin@v2.0.1
-        id: fc
-        with:
-          issue-number: ${{ github.event.number }}
-          body-includes: ${{ env.check-type }}
-      - uses: thollander/actions-comment-pull-request@686ab1cab89e0f715a44a0d04b9fdfdd4f33d751 # pin@v1.4.1
-        if: >
-          (needs.check-semantic-pr.result == 'failure')
-          || (steps.fc.outputs.comment-id != '')
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+        if: always()
+      - uses: ./.github/workflows/composite/comment-on-pr
+        if: always()
+        env:
+          check-type: Semantic PR check
+          check-documentation: See [instructions on formatting your commit and pull request titles](https://github.com/magma/magma/wiki/Contributing-Code#pull-request-and-commit-message-title-are-following-conventional-commits-format).
```

Comment:
Might be with `>`. However, would require more testing than is worth the effort, as this would mix yaml features with markdown requirements. Is out of scope for me, atm.